docs/tasks/run-application/update-api-object-in-place.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Update API Objects in Place Using kubectl patch
```

Comment:
Currently, [Managing Resources](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/) is a mix of conceptual material and several tasks. I'l like to separate out the tasks into their own topics. Maybe under Tasks, we'll want to have a bucket for tasks about updating.